plugin/src/main/java/at/helpch/chatchat/command/MentionToggleCommand.java

```diff
@@ -0,0 +1,33 @@
+package at.helpch.chatchat.command;
+
+import at.helpch.chatchat.ChatChatPlugin;
+import at.helpch.chatchat.api.ChatUser;
+import dev.triumphteam.cmd.bukkit.annotation.Permission;
+import dev.triumphteam.cmd.core.BaseCommand;
+import dev.triumphteam.cmd.core.annotation.Command;
+import dev.triumphteam.cmd.core.annotation.Default;
+
+@Command(value = "togglemention", alias = "toggleping")
+public class MentionToggleCommand extends BaseCommand {
+
+    private static final String TOGGLE_PERMISSION = "chatchat.mentiontoggle";
```

Comment:
Please add the permission in build.gradle as well. thanks

Reply:
Done